src/services/vault.py

```diff
@@ -0,0 +1,40 @@
+import os, pathlib
+import sys
+
+# Added to make consistent import paths with respect to src
+sys.path.append(f"{pathlib.Path(__file__).parent.resolve()}/../..")
+from src.services.selenium.platforms import Platform
+import constants
```

Comment:
You don't need to have constants below the `path.append`, python just treats it as a globally available file that can be imported from anywhere since it is present in the root. This can be shifted on line 1.

Reply:
It gives me a ModuleNotFound error when I move it above the `path.append`